Antidote/AvatarsManager.m

```diff
@@ -98,6 +98,44 @@ - (UIImage *)createAvatarFromString:(NSString *)string diameter:(CGFloat)diamete
     return [self imageWithView:label];
 }
 
+- (UIImage *)createAvatarFromString:(NSString *)string diameter:(CGFloat)diameter
+                          textColor:(UIColor *)textColor backgroundColor:(UIColor *)backgroundColor
```

Comment:
To avoid code duplication you should refactor `createAvatarFromString:diameter:` to call this method with colors from `appearance`